api/src/main/java/io/opentelemetry/trace/StatusCanonicalCode.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.trace;
+
+/**
+ * The set of canonical status codes. If new codes are added over time they must choose a numerical
+ * value that does not collide with any previously used value.
+ *
+ * @since 0.1.0
+ */
+public enum StatusCanonicalCode {
+
+  /**
+   * The operation has been validated by an Application developers or Operator to have completed
+   * successfully.
+   */
+  OK(0),
```

Comment:
The spec doesn't seem to define numeric values for the codes, do we need these?

https://www.github.com/open-telemetry/opentelemetry-specification/tree/master/specification%2Ftrace%2Fapi.md

Reply:
Filed https://github.com/open-telemetry/opentelemetry-java/issues/1744 to track this.